Scout.py

```diff
@@ -1,39 +1,8 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
We started removing those since it's implicit with the most recent versions of python.
``suggestion
```